Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:20-alpine
 
-RUN apk add --no-cache curl=8.8.0-r0 python3=3.12.3-r1 && \
+RUN apk add --no-cache curl=8.9.0-r0 python3=3.12.3-r1 && \
```

Comment:
This change is unrelated to the feature, but was required to get the image building again.